prover/src/lib.rs

```diff
@@ -0,0 +1,83 @@
+#![allow(incomplete_features)]
+#![feature(generic_const_exprs)]
+
+use std::time::Instant;
+
+use sp1_core::{
+    runtime::Program,
+    stark::{Proof, RiscvAir, StarkGenericConfig, VerifyingKey},
+    utils::{run_and_prove, BabyBearPoseidon2},
+};
+use sp1_recursion_core::runtime::Runtime;
+use sp1_recursion_program::compress::build_compress;
+
```

Comment:
Eventually, I think it would be nicer to have these methods on a struct to make it more object oriented.